srcpkgs/sane/template

```diff
@@ -1,28 +1,43 @@
 # Template file for 'sane'
 pkgname=sane
-_gitlab_release_hash=8bf1cae2e1803aefab9e5331550e5d5d
-version=1.0.31
-revision=3
+_gitlab_release_hash=104f09c07d35519cc8e72e604f11643f
+version=1.0.32
+revision=1
 wrksrc="sane-backends-${version}"
 build_style=gnu-configure
 configure_args="--disable-locking --enable-ipv6 --enable-pthread
  --with-usb --docdir=/usr/share/doc/sane ac_cv_func_mmap_fixed_mapped=yes
  $(vopt_with snmp)"
 hostmakedepends="pkg-config"
-makedepends="avahi-libs-devel libjpeg-turbo-devel tiff-devel libgphoto2-devel
- v4l-utils-devel libusb-devel openssl-devel libxml2-devel
- $(vopt_if snmp net-snmp-devel)"
+makedepends="libjpeg-turbo-devel tiff-devel libgphoto2-devel v4l-utils-devel
+ libusb-devel openssl-devel libxml2-devel $(vopt_if snmp net-snmp-devel)
+ $(vopt_if avahi avahi-libs-devel)"
 depends="$(vopt_if snmp net-snmp)"
 conf_files="/etc/sane.d/*.conf"
 short_desc="Scanner Access Now Easy"
 maintainer="Piraty <piraty1@inbox.ru>"
 license="GPL-2.0-or-later, SANE-exception"
 homepage="http://www.sane-project.org/"
+changelog="https://gitlab.com/sane-project/backends/-/raw/master/NEWS"
 distfiles="https://gitlab.com/sane-project/backends/uploads/${_gitlab_release_hash}/sane-backends-${version}.tar.gz"
-checksum=4a3b10fcb398ed854777d979498645edfe66fcac2f2fd2b9117a79ff45e2a5aa
+checksum=3a28c237c0a72767086202379f6dc92dbb63ec08dfbab22312cba80e238bb114
 noshlibprovides="avoid false detection of device drivers"
 
-build_options="snmp"
+# additional group 'lp' is required by saned to access some all-in-one devices
+system_accounts="_saned"
+_saned_groups="lp,scanner"
+
+build_options="avahi snmp"
+build_options_default="avahi"
+
+# enable support for escl (airscan) backend
+if [ "$build_option_avahi" ]; then
+	makedepends+=" libcurl-devel"
+fi
```

Comment:
add this to `vopt_if avahi` , leave a comment somewhere like `escl only available if avahi is enabled` (at least in the commit, not necessarily in the template)